exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -344,3 +348,25 @@ func (prwe *PrwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequ
 	}
 	return nil
 }
+
+// FormatHeaderValues will add the name/version pair to the User-Agent request
+// header. If the extra parameters are provided they will be added as metadata to the
+// name/version pair resulting in the following format.
+// "name/version (extra0; extra1; ...)"
+// The User-Agent part will be concatenated with this current request's User-Agent string.
+func FormatHeaderValues(req *http.Request, name, version string, extra ...string) {
+	ua := fmt.Sprintf("%s/%s", name, version)
+	if len(extra) > 0 {
+		ua += fmt.Sprintf(" (%s)", strings.Join(extra, "; "))
```

Comment:
```suggestion
		ua = fmt.Sprintf("%s (%s)", ua, strings.Join(extra, "; "))
```

Feels like this is slightly clearer about intent.